README.md

```diff
@@ -103,96 +97,31 @@ For webhooks to work, the server must be publicly accessible on the internet. Fo
-```
+This sample app can also demonstrate the creation of a processor token for use with Plaid partners. While still initializing Link with the Auth product, instead of of using Plaid Auth endpoints, an example of the creation of a processor token and integration with a plaid partner is included in the [root items route][items-routes].
 
-with this line instead:
+#### Using Dwolla sandbox as a test case
 
-```bash
-while [ "$(curl -s -o /dev/null -w "%{http_code}" -m 1 https://localhost:3002)" != "200" ]
-```
+To test transferring of funds with a Plaid Partner, this sample app uses Dwolla sandbox. Please follow these steps:
 
-After starting up the Pattern sample app, you can now view it at https://localhost:3002. Your browser will alert you with an invalid certificate warning; click on "advanced" and proceed.
+1. Go to your [Plaid developer dashboard team settings/integrations page](https://dashboard.plaid.com/team/integrations). Scroll down to Dwolla and click on the "enable" button.
 
-### Working with Plaid Partners
+2. Head to the [Dwolla site](https://developers.dwolla.com/guides/sandbox) and create a sandbox account by clicking the `Get API Keys` button and following the sign up instructions.
 
-This sample app can also demonstrate the creation of a processor token for use with Plaid partners. While still initializing Link with the Auth product, instead of of using Plaid Auth endpoints, an example of the creation of a processor token is included in the [root items route][items-routes].
+3. After logging in, create a temporary access token by clicking on the "Create Token" button on Dwolla's dashboard [Applications Page](https://dashboard-sandbox.dwolla.com/applications-legacy). Copy this token and add it to the .env file as `DWOLLA_ACCESS_TOKEN`. This is a temporary access token issued by Dwolla which is good for one hour.
+4. Go to your Dwolla [Master Account Funding Source](https://dashboard-sandbox.dwolla.com/account/funding-sources) on the dashboard and obtain your id from your Superhero Savings Bank. This is the very last number in the smaller white box on the bottom left corner of the page. Enter this id in the .env file as the `DWOLLA_MASTER_ACCOUNT_ID`.
+5. Finally, make sure that `IS_PROCESSOR` is set to `true` in the .env file.
 
 ### Verifying and transferring funds
 
-This sample app demonstrates how to get the available balance in order to verify funds. It does not actually conduct a transfer of funds. Therefore, the balance in the linked account (whether in sandbox or development) will not decrement when a transfer is made in this app.
+This sample app demonstrates how to get the available balance in order to verify and transfer funds. However, it does not actually conduct a transfer of funds because it uses Dwolla sandbox. Therefore, the balance in the linked account (whether in sandbox or development) will not decrement when a transfer is made in this app.
```

Comment:
```suggestion
This sample app demonstrates how to get the available balance in order to verify and transfer funds. However, it does not actually conduct a transfer of funds because it uses Dwolla sandbox. The balance in the linked account (whether in sandbox or development) will not decrement when a transfer is made in this app.
```